### PR TITLE
Fix broken facility link in the _user_approvals partial

### PR DESCRIPTION
**Pull request by vkrmis**
https://www.pivotaltracker.com/story/show/170820401

The `_user_approvals` partial had a non working link to the user's facility page. This PR uses the `admin_facility_group_facility_path` to link to the facility's page.

### CODE REVIEW

File: app/views/shared/_user_approvals.html.erb [hunk 1]
```diff
@@ -9,18 +9,33 @@
...
                 <div class="text-grey mt-1 mb-2">
                     <% if user.sync_approval_status_reason.present? %>
-                      <span><%= user.sync_approval_status_reason %> at <%= user.registration_facility.present? ? link_to(user.registration_facility.name, admin_facilities_path(user.registration_facility)) : "N/A" %></span>, <%= l user.updated_at.to_date %>
+                      <span><%= user.sync_approval_status_reason %> at
+                        <% if user.registration_facility.present? %>
+                          <%= link_to(user.registration_facility.name,
+                                      admin_facility_group_facility_path(user.registration_facility.facility_group,
+                                                                        user.registration_facility)) %>
+                        <% else %>
+                          <%= "N/A" %>
```
Comment (harimohanraj89) [inline app/views/shared/_user_approvals.html.erb]: This doesn't need to be in ERB tags.

File: app/views/shared/_user_approvals.html.erb [hunk 2]
```diff
@@ -9,18 +9,33 @@
...
 
-                    <%= link_to "Allow access", admin_user_enable_access_path(user), method: :put, class: "btn btn-sm btn-outline-success", data: { confirm: I18n.t('admin.enable_access_alert') } %>
-                    <%= link_to '<i class="fas fa-times"></i>'.html_safe, '#deny-access-modal-' + user.id, 'data-toggle' => 'modal', class: "btn btn-sm btn-outline-danger" %>
+                    <%= link_to "Allow access", admin_user_enable_access_path(user),
+                                method: :put,
+                                class: "btn btn-sm btn-outline-success",
+                                data: { confirm: I18n.t('admin.enable_access_alert') } %>
+                    <%= link_to '<i class="fas fa-times"></i>'.html_safe, '#deny-access-modal-' + user.id,
+                                'data-toggle' => 'modal',
+                                class: "btn btn-sm btn-outline-danger" %>
```
Comment (harimohanraj89) [inline app/views/shared/_user_approvals.html.erb]: I guess these chunks are just reformatting?

File: app/views/shared/_user_approvals.html.erb (hunk 1 above)
Comment (kitallis) [reply app/views/shared/_user_approvals.html.erb]: Fixing this so we can merge it in.

File: app/views/shared/_user_approvals.html.erb (hunk 2 above)
Comment (harimohanraj89) [reply app/views/shared/_user_approvals.html.erb]: Not a big deal. Resolving.

**Comment by harimohanraj89** [comment]
Looks great @kitallis. :shipit:

**Pull request merged**